Automattic/wp-calypso
# Create AB Test for Monthly Prices in Add Domain Flow G Suite Prompt

#### Changes proposed in this Pull Request

* Slight Refactor of `GoogleAppsProductDetails`
* AB test for showing Monthly Prices for users using `USD`: `gsuiteDomainFlowOptions`
   * `hideMonthlyPrice` - Only show the price per user per year
   * `showMonthlyPrice` - Show the price per month per user and the annual price

**hideMonthlyPrice**
<img width="742" alt="Screen Shot 2019-04-11 at 1 35 26 PM" src="https://user-images.githubusercontent.com/2810519/55991891-175b8380-5c60-11e9-8381-f926d97dad2f.png">


**showMonthlyPrice**
<img width="743" alt="Screen Shot 2019-04-11 at 1 35 37 PM" src="https://user-images.githubusercontent.com/2810519/55991889-162a5680-5c60-11e9-9629-c90b602eacd5.png">

#### Testing Instructions

**TIP:** IF you are having issues with the abtests info can be printed to console by running `localStorage.setItem( 'debug', 'calypso:abtests' )`

1. Make sure you use a test account using `USD`
1. Begin Adding a domain to a site via the sidebar
1. Reach the "Add Professional email from G Suite by Google Cloud to :domain` step
1. Confirm you were assigned a variation in the `gsuiteDomainFlowOptions`  test.
1. Enter each variation of the A/B Test mentioned above ( can be set from the "bug" menu in the lower right )  <img width="600" alt="Screen Shot 2019-04-11 at 1 35 42 PM" src="https://user-images.githubusercontent.com/2810519/55991815-014dc300-5c60-11e9-8058-5bba90a85a46.png">
1. Make sure the screen matches the sceenshot above
